SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+If you are going to contribute to this project, please take some time out to read through this. Follow the set rules and NO exceptions. 
+
+## Reporting a Vulnerability
+
+To report a vulnerability, create a new topic in the Issues. Be straight to the point and precise. 
+
+Bugs to report:
+
+|   |   |
+|:-:|---|
+|   |   |
+|   |   |
+|   |   |
+|   |   |
+|   |   |
+
+This is a completely crowd sourced project and thus, there is no bug bounty. 
```

Comment:
i’d add (yet).  being open source doesn’t preclude getting paid for development, a la mastodon’s setup. 

take a look at the CONTRIBUTING section of their readme: https://github.com/tootsuite/mastodon

Reply:
Roger that. Maybe its something that we can keep in view for now instead of completely disregarding the possibility. I have updated the file accordingly.